frontend/src/components/samples/SampleEditContent.js

```diff
@@ -29,6 +29,9 @@ import { requiredRules, nameRules } from "../../constants";
 const searchSamples = (token, input) =>
   withToken(token, api.samples.search)(input).then(res => res.data.results)
 
+const searchCoordinates = (token, input, options) =>
```

Comment:
There are a couple of issues with this.
1. If you edit an existing sample, the Select element shows the coordinate ID when the form is first displayed. It only switches to showing a coordinate name if the user clicks on the field to initialize the coordinate options.
2. `options` is undefined and the api defaults to returning 100 coordinates, from A00 to D22. This doesn't seem to match the coordinate choices for a 96 well plate or a 384 well plate.

* I think the select options should be based on the container type.
* We should initialize the coordinate options when the component is first rendered, rather than on each `onSearch` call. 
* I would disable the coordinates field if the container type hasn't been selected yet.